### ISSUE TITLE
Add methods for observed information matrices

### ISSUE THREAD

**Issue by jepusto** [jepusto/lmeInfo#25]
This will require calculating 2nd derivatives of error variance-covariance components.

**Comment by jepusto** [comment]
Correct, it's not yet implemented. It will take some further work because
the observed information requires second derivatives of the
variance-covariance matrix (expected and average information don't). We can
add a note in the docs and error checking in the function to make clear
which are supported.

**Comment by wviechtb** [comment]
A note would be useful. I played around with it for a while (thinking that maybe it was the particular model I was working with where it didn't work) and was headed here to make a bug report when I saw this issue.
